### PR TITLE
Setup and initial sections

### PR DESCRIPTION
**Pull request by BahirHakimy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/30e1c676d3.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <main>
+      <header>
+        <strong>Welcome</strong>
```
Comment (brainconnect93) [inline index.html]: - [ ] I would kindly suggest you make this an `anchor tag`. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. 👍 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <i class="fa-solid fa-bars"></i>
```
Comment (brainconnect93) [inline index.html]: - [ ] I would kindly suggest you wrap your `hamburger` icon inside a `nav` tag, The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+          <div class="social-icon">
+            <i class="fa-brands fa-linkedin"></i>
+          </div>
+          <div class="social-icon">
+            <i class="fa-brands fa-whatsapp"></i>
+          </div>
+          <div class="social-icon">
+            <i class="fa-solid fa-envelope"></i>
+          </div>
+        </div>
```
Comment (brainconnect93) [inline index.html]: - [ ] I would kindly suggest the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. 👍 👍

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :recycle:

### Hi @BahirHakimy, 👋 

Good job so far, You really made a great effort working on your project, it looks really awesome! but there are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @BahirHakimy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [ ] Implemented all requested changes from the previous reviewer :heavy_check_mark: 
- [ ] Good UI design :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**